lms/static/scripts/ui-playground/components/ErrorComponents.tsx

```diff
@@ -466,7 +466,8 @@ export default function ErrorComponents() {
             <div className="grid grid-cols-2 gap-2">
               {(
                 [
-                  'd2l_file_not_found_in_course',
+                  'd2l_file_not_found_in_course_student',
```

Comment:
@lyzadanger Is this all it's needed to fix them for the ui-playground? 

I don't know how to check that locally.

Reply:
It's easy! Go here: http://localhost:8001/ui-playground/errors and find the error you want to look at: 

<img width="802" alt="image" src="https://user-images.githubusercontent.com/439947/226638530-e79c677e-0ae4-4de4-ac05-98b0f8f1d2c3.png">